<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-077G3EM" failures="0" tests="18" name="com.MlwalletScripts.MlwalletScripts" time="0.473" errors="9" timestamp="2022-09-16T14:51:15 IST" skipped="9">
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="MLWalletAppLaunch" time="0.000">
    <skipped/>
  </testcase> <!-- MLWalletAppLaunch -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="MLWalletAppLaunch" time="0.444">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- MLWalletAppLaunch -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="Login" time="0.000">
    <skipped/>
  </testcase> <!-- Login -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="Login" time="0.004">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Login -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="SendMoney" time="0.004">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- SendMoney -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="SendMoney" time="0.000">
    <skipped/>
  </testcase> <!-- SendMoney -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WalletToWallet" time="0.000">
    <skipped/>
  </testcase> <!-- WalletToWallet -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WalletToWallet" time="0.002">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- WalletToWallet -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="KwartaPadala" time="0.000">
    <skipped/>
  </testcase> <!-- KwartaPadala -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="KwartaPadala" time="0.003">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- KwartaPadala -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WithdrawMoney" time="0.003">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- WithdrawMoney -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WithdrawMoney" time="0.000">
    <skipped/>
  </testcase> <!-- WithdrawMoney -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="RecievedMoney" time="0.006">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- RecievedMoney -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="RecievedMoney" time="0.000">
    <skipped/>
  </testcase> <!-- RecievedMoney -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WalletToWalletNegativeScenario" time="0.004">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- WalletToWalletNegativeScenario -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WalletToWalletNegativeScenario" time="0.000">
    <skipped/>
  </testcase> <!-- WalletToWalletNegativeScenario -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WrongOTPHandle" time="0.000">
    <skipped/>
  </testcase> <!-- WrongOTPHandle -->
  <system-out/>
  <testcase classname="com.MlwalletScripts.MlwalletScripts" name="WrongOTPHandle" time="0.003">
    <error message="nodeName cannot be null or empty" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: nodeName cannot be null or empty
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:166)
at com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:274)
at com.extent.ExtentReporter.HeaderChildNode(ExtentReporter.java:251)
at com.extent.ExtentReporter.onTestSkipped(ExtentReporter.java:241)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:57)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeListenersForSkippedTestResult(TestInvoker.java:446)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:542)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- WrongOTPHandle -->
  <system-out/>
</testsuite> <!-- com.MlwalletScripts.MlwalletScripts -->
